app/Domain/Integrations/Models/IntegrationModel.php

```diff
@@ -51,6 +51,21 @@ final class IntegrationModel extends UuidModel
         'partner_status' => IntegrationPartnerStatus::THIRD_PARTY,
     ];
 
+    public function canBeActivated(): bool
+    {
+        return $this->status !== IntegrationStatus::Active->value;
```

Comment:
`return $this->status !== IntegrationStatus::Active->value && $this->status !== IntegrationStatus::Blocked->value;`

If you block an `Integration` in `Draft`, you still see the option to `activate` it, which seems counterintuitive to me.

Reply:
Do you mean that you can `Block` or `Activate` an integration that has the status `Draft`?

Reply:
Blocking a `Draft` is `OK`. But should it be possible to `Activate` an `Integration` that has been `Blocked`. 
I assume this shouldn't be possible.

Reply:
I added it because you can `Block` an Integration by "mistake", so now you have the option to `Activate` it again.

Reply:
Oh, that might indeed be a good idea. Though, I wonder about the flow. If an Integration was activated with a `coupon` and accidentally blocked, you could not reactivate it with the same `coupon`.
But maybe this could be tackled in another ticket?

Reply:
Indeed, very good point. Maybe this needs an extra tweak and needs to become a reactivate without the need to enter a coupon or organization details?